coverage/.resultset.json

```
@@ -0,0 +1,437 @@
+{
+  "RSpec": {
+    "coverage": {
+      "/Users/aloconnor/git/danger-spelling/lib/danger_plugin.rb": {
```

```suggestion
      "/home/runner/work/danger-spelling/danger-spelling/lib/danger_plugin.rb": {
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>